### PR TITLE
feat: add alt for navigation logo, focusable props for dropdown and lnks 

### PR DESCRIPTION
**Pull request by Pavel-Tyan**
Task: https://github.com/orgs/diplodoc-platform/projects/2/views/1?pane=issue&itemId=70132343

1. Make all navigation items focusable
2. Add aria-expanded for drop-down list
3. Add default text for logo image

Before: 

https://github.com/user-attachments/assets/031d0582-882b-4ec8-b2fd-9d7d85155368

After:

https://github.com/user-attachments/assets/548c7b82-a718-4bed-b99a-7d35b71f5ce6

### CODE REVIEW

File: src/navigation/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -14,13 +14,16 @@ const b = block('logo');
 
 export type LogoProps = ThemedNavigationLogoData & {
     className?: string;
+    alt?: string;
 };
 
-const Logo: React.FC<LogoProps> = (props) => {
+const Logo: React.FC<LogoProps> = ({alt = 'Logo', ...restProps}) => {
```
Comment (NikitaCG) [inline src/navigation/components/Logo/Logo.tsx:20]: we use `i18`, and I think we need translation for alt in this place, you can see example in `src/components/Security`

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/navigation/components/Logo/Logo.tsx (hunk 1 above)
Comment (Pavel-Tyan) [reply src/navigation/components/Logo/Logo.tsx]: Added translation

**Review by NikitaCG — APPROVED** [review]

**Comment by NikitaCG** [comment]
@Pavel-Tyan Hi, you need to fix tests and lints))

**Comment by Pavel-Tyan** [comment]
> @Pavel-Tyan Hi, you need to fix tests and lints))
Fixed problems with typization that affect tests and lints.

**Review by NikitaCG — DISMISSED** [review]

**Comment by NikitaCG** [comment]
I think we need to add prop alt in `NavigationLogoData` also

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**